clang/lib/DPCT/PatternRewriter.cpp

```diff
@@ -719,8 +728,50 @@ void setFileTypeProcessed(enum SourceFileType FileType) {
   SrcFileType = FileType;
 }
 
+static void constructWaringMsg(const std::string &Input, size_t index,
+                               const std::string &FileName,
+                               const std::string &FrontPart,
```

Comment:
instead of passing frontpart as a string, can we pass the line number of frontpart instead?

Reply:
Yes, we can. Passing the Frontpart string makes it easy to identify the root cause of a possible incorrect line number. So, I suggest keeping the current style.